modules/mobile/spec/requests/mobile/v0/user/addresses_spec.rb

```diff
@@ -262,7 +272,7 @@
 
   describe 'POST /mobile/v0/user/addresses/validate', :skip_va_profile_user do
     let(:address) do
-      address = build(:va_profile_address, vet360_id: user.vet360_id)
+      address = build(:va_profile_address, :v2_override)
```

Comment:
This should fail. va_profile_address is a factory for the VAProfile::Address model and not VAProfile::V3::Adress.